src/pages/ListPage/SpaceList/Building/Building.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Building({ space }) {
```

Comment:
```suggestion
function Building({ space }) {
   const {title, city, ~~~~} = space;
}
```
구조분해할당 한번 더해서 쓰면 깔끔하겠네요!

Reply:
구조 분해 할당 후 적용시켜줬습니다!